tests/unit/30_core_1.bats

```diff
@@ -128,6 +128,16 @@ build_nginx_config() {
   assert_success
 }
 
+@test "(core) run command from Procfile" {
+  deploy_app
+  run /bin/bash -c "dokku run $TEST_APP console -c 'echo hi' | tail -n 1"
+  echo "output: "$output
+  echo "status: "$status
+
+  assert_success
+  assert_output "hi"
```

Comment:
This tests is currently failing because of an issue I ran into with how herokuish handles arguments.

Observe this:

```sh
root@dokku:/home/vagrant# docker run --label=dokku -i -t dokku/nodejs:latest /exec bash -c 'echo hi'

root@dokku:/home/vagrant#
```

```sh
root@dokku:/home/vagrant# docker run --label=dokku -i -t dokku/nodejs:latest bash -c 'echo hi'
hi
root@dokku:/home/vagrant#
```

Could this be a bug in herokuish?